packages/expect/src/__tests__/matchers.test.js

```diff
@@ -1076,6 +1333,24 @@ describe('.toBeCloseTo()', () => {
     });
   });
 
+  if (isBigIntAllowed) {
+    [
+      asBigInt(0, 0),
+      [MAX_SAFE_AS_BIGINT, MAX_SAFE_AS_BIGINT + BigInt(49)],
+      [BigInt(0), 49],
```

Comment:
@pedrottimark 
> I think we need parallel `} else if (typeof received === 'bigint' && typeof expected === 'bigint) {'
> 
> and then final `else { … }` for mixed number and bigint to throw matcher error

So you're suggesting not to allow mixed for `tobecloseto` ? This would be the only numeric matcher to throw, unless your suggesting that for the others as well.

Reply:
I think that's what Mark is suggesting, see his quote:

> I think toBeCloseTo would be limited to same type for received and expected, both number or both bigint, because receivedDiff is computed by subtraction.

I was originally against allowing mixing at all, but I'll trust Mark's judgement on this one since he does most of the work on matchers :sweat_smile:

Reply:
Oh ok, that makes sense then if it were to be the only one then. But the reason i would suggest allowing mixed, is keeping it consistent with the others.
```ts
function someFunc(someArg):Array<bigint | number>

✔️test('all of someFuncs resulting values are greater than 10',()=>{
someFunc(someArg).forEach(value: number | bigint => expect(value).toBeGreaterThan(10))
})

✔️test('all of someFuncs resulting values are less than 100',()=>{
someFunc(someArg).forEach(value: number | bigint => expect(value).toBeLessThan(100))
})

🛑 test('all of someFuncs resulting values are close to 50',()=>{
someFunc(someArg).forEach(value: number | bigint => expect(value).toBeCloseTo(50))
}) 

//Would need to do a check..
✔️test('all of someFuncs resulting values are close to 50',()=>{
someFunc(someArg).forEach(value: number | bigint => expect(value).toBeCloseTo(typeof value ==='number' ?50: Bigint(50)))
})
```

Reply:
removed bigint support for tobeclosto